app/models/time_range.rb

```diff
@@ -0,0 +1,29 @@
+class TimeRange
+  FORMAT = "%l:%M %P".freeze
+
+  attr_reader :start_at, :end_at
+
+  def initialize(start_at, end_at)
+    @start_at = start_at
+    @end_at = end_at
+  end
+
+  def to_s
+    I18n.t(
+      "time_ranges.format",
+      weekday: weekday.label,
+      start_at: format(start_at),
+      end_at: format(end_at),
+    ).squish
```

Comment:
Why `squish`? What does it look like before squishing, and what does it look like after?